CONTRIBUTING.md

```diff
@@ -65,6 +65,20 @@ clang-format style=file <your file>
 
 Refer to [ClangFormat documentation](https://clang.llvm.org/docs/ClangFormat.html) for more information.
 
+### editorconfig-checker
+
+We also recommend using [editorconfig-checker](https://github.com/editorconfig-checker/editorconfig-checker) to ensure your code adheres to the project's coding style. EditorConfig-Checker is a command-line tool that checks your code against the rules defined in the [.editorconfig](https://github.com/oneapi-src/oneDAL/blob/main/.editorconfig) file.
```

Comment:
.editorconfig does not exist in /blob/main so this shows up as a broken link.k

Reply:
Not yet, but it would exist there after this is merged